### PR TITLE
Branch user guide

### PR DESCRIPTION
**Pull request by ngdeqi**

### CODE REVIEW

**Review by zoranabc201 — COMMENTED** [review]
Looks good.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,33 +1,92 @@
...
+    * [Delete Client: `delete -client`](#delete-client-delete--client)
+    * [List Clients: `list -client`](#list-clients-list--client)
+    * [Check Client: `check -client`](#check-client-check--client)
+    * [Pair Client and Property: `pair`](#pair-client-and-property-pair)
+    * [Unpair Client and Property: `unpair`](#unpair-client-and-property-unpair)
+    * [Exit: `quit`](#exit-quit)
+    * [Saving data](#saving-data)
+    * [Loading data](#loading-data)
+* [FAQ](#faq)
+* [Command Summary](#command-summary)
```
Comment (wilsonngja) [inline docs/UserGuide.md:23]: Good inclusion of the content page.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,33 +1,92 @@
...
+
+
+### List Properties: `list -property`
+
+### Check Property: `check -property`
+Displays the information of the specified property, along with the clients the property is being rented by, if any.
+
+<u>Format</u>: `check ip/PROPERTY_INDEX`
+
+<u>Example</u>: `check ip/2`
```
Comment (wilsonngja) [inline docs/UserGuide.md:51]: I like how you included examples together with the format.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,33 +1,92 @@
...
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+<u>Format</u>: `pair ip/PROPERTY_INDEX ic/CLIENT_INDEX`
+* `PROPERTY_INDEX` and `CLIENT_INDEX` must be indexes present in the client list and property list respectively.
+* The client must not currently be in a pairing.
+* The property can be paired with multiple clients.
+
+<u>Example</u>: `pair ip/1 ic/5`
+
+### Unpair Client and Property: `unpair`
+Unpairs the client from the specified property, to record that the client is no longer renting the property.
```
Comment (wilsonngja) [inline docs/UserGuide.md:77]: It's really good that you explain in greater details about the feature.

**Review by wilsonngja — APPROVED** [review]
A very good effort on the User Guide. It explains clearly what the feature does and I really like the explanation given.

**Pull request merged**